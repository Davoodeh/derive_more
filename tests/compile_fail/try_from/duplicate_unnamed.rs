struct F1;

#[derive(derive_more::TryFrom)]
#[try_from(usize)]
enum Enum {
    Field(F1),
    Field2(F1),
}

fn main() {}
