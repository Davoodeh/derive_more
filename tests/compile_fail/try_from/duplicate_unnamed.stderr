error: `F1` types collection is used for more than one variant (`Field`, `Field2`), non-repr `try_from` cannot be implemented (try changing the order of fields as a workaround)
 --> tests/compile_fail/try_from/duplicate_unnamed.rs:7:11
  |
7 |     Field2(F1),
  |           ^^^^
